src/olympia/files/models.py

```diff
@@ -706,6 +712,50 @@ class Meta:
         db_table = 'validation_annotations'
 
 
+class WebextPermission(ModelBase):
+    KNOWN_PERMISSIONS = WEBEXT_PERMISSIONS_DICT
+    name = models.CharField(max_length=255, unique=True, null=False)
+    _enum = models.IntegerField(default=WEBEXT_UNKNOWN.id, db_column='enum')
```

Comment:
Bonus: we wouldn't even need object constants (that are not terribly useful) with this idea.